Cim2pp: test to check dtypes of columns

lezneew opened the pull request:
cim2pp converter: added test to check standard dtypes of pandapower net element columns

Commit:
cim2pp: refactored code

heckstrahler committed:
Merge branch 'develop' into cim2pp_WSG

Commit:
Merge branch 'develop' of github.com:lezneew/pandapower into develop

Commit:
Merge branch 'develop' into cim2pp_WSG

Commit:
cim2pp: added tests for node breaker model

Commit:
cim2pp: refactored code

Commit:
cim2pp: added node breaker test net to pandapower repo

Commit:
cim2pp: added node breaker test net to pandapower repo

heckstrahler committed:
Update CHANGELOG.rst

lezneew committed:
Merge branch 'e2nIEE:develop' into cim2pp_WSG

lezneew committed:
cim2pp converter: added test to check standard dtypes of pandapower net element columns

lezneew committed:
fixed changelog

lezneew committed:
Merge branch 'e2nIEE:develop' into cim2pp_WSG

vogt31337 commented:
@lezneew have a look at PR #2533, there the author setup a way to check the dtypes against an empty network.

lezneew committed:
added dynamic test to check dtypes of network element columns

lezneew committed:
added typing for test networks

lezneew committed:
Merge branch 'develop' into cim2pp_WSG

vogt31337 committed:
Update test_from_cim.py

moved from general import to specific ones.

vogt31337 committed:
Merge branch 'develop' into cim2pp_WSG

vogt31337 committed:
Update test_from_cim.py

added a "result" which shows what is the expected result, only if differences are found.

vogt31337 committed:
Update test_from_cim.py

modified description parameter in case of an dtype error, so that it will always stay on top.

lezneew committed:
cim2pp: refactored code

lezneew committed:
cim2pp: refactored CHANGELOG.rst

heckstrahler committed:
Merge branch 'develop' into cim2pp_WSG

lezneew committed:
Merge branch 'develop' into cim2pp_WSG

# Conflicts:
#	CHANGELOG.rst
#	pandapower/test/converter/test_from_cim.py

lezneew committed:
cim2pp: refactored code to work with dtype tables directly

heckstrahler commented:
maybe moving to pandera validation is the better longterm see #2717
